packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -317,5 +317,8 @@ export const comments: { readonly [key in TokenName]: string } = {
     calendarEventFillBlockedColor:
         'Background fill to indicate the occupied slots in calendar resource rows',
     calendarGrabHandleBackgroundColor:
-        'Background fill color for the editable calendar event grab handle'
+        'Background fill color for the editable calendar event grab handle',
+    calendarGridBorderColor: 'Border color for the calendar grid',
+    calendarParentRowBackgroundColor:
```

Comment:
instead of part: parent row, can we call this part: group header? `calendarGroupHeaderBackgroundColor`
Seems to be analogous to the table group header concept:
![image](https://github.com/user-attachments/assets/7545a308-7e68-4aa5-a2c0-2baec0c3d8fe)

At least a concept more specific than "parent row"

Reply:
I agree. Updated the part to: "group header".